examples/react/next-js-app/pages/index.tsx

```diff
@@ -52,10 +52,10 @@ const Home: NextPage = () => {
                             <span> Your feature-release variable value = {JSON.stringify(variable?.value)} </span>
                         </div>
                         <div>
-                            <span> Your variableString variable = {variableString?.value} </span>
```

Comment:
aren't you changing what `variable` object is being accessed here?

Reply:
oopsies